gp-templates/project-import.php

```diff
@@ -24,9 +24,17 @@
 		$format_options[$slug] = $format->name;
 	}
 	$format_dropdown = gp_select( 'format', $format_options, 'auto' );
+
 ?>
-	<dt><label	for="format"><?php _e( 'Format:', 'glotpress' ); ?></label></dt>
+	<dt><label for="format"><?php _e( 'Format:', 'glotpress' ); ?></label></dt>
 	<dd><?php echo $format_dropdown; ?></dd>
+	<?php if ( $can_import && ! $can_only_import_waiting ): ?>
```

Comment:
> Notice: Undefined variable: can_import in /gp-templates/project-import.php on line 31

The template is also used to import originals. In this case we don't have to select a status and should instead hide it completely.